platform/lifecycle_inputs.go

```diff
@@ -341,6 +346,13 @@ func ValidateOutputImageProvided(i *LifecycleInputs, logger log.Logger) error {
 	return nil
 }
 
+func ValidateMultiplesTargetsProvided(i *LifecycleInputs, _ log.Logger) error {
```

Comment:
```suggestion
func ValidateSingleTargetProvided(i *LifecycleInputs, _ log.Logger) error {
```
I think this is more accurate as to what we are "validating"

Reply:
I think this will pose some problems if we don't know if the input image is daemon, registry, or layout. How will the lifecycle know where to look for it?